### PR TITLE
Docker update

### PR DESCRIPTION
**Pull request by hansvancalster**

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -10,27 +10,24 @@ After review of a protocol-branch and before each release the following steps wi
...
-2.  A second GitHub Action will be run when a pull request to be merged into the main branch *is approved*.
-
-    1.  This action will automatically update:
+    3.  This action will also automatically update:
 
         1.  the repo-wide `NEWS.md` file with `protocolhelper:::update_news_release()`
 
         2.  the `.zenodo.json` file with `protocolhelper:::update_zenodo()`
 
-    1.
```
Comment (ElsLommelen) [inline RELEASES.md:27]: Comment on line below this one: this is not the second Github Action, it is incorporated in the first one.  So check the lines below as well.  Or you may want to move up the actions above this indicated line (lines 19-24) and put them just below the other steps of the GHA (line 12).  After this, you just need one of the texts describing what to do (either lines 13-17 or lines 25-28), or you can mix them together and keep the most relevant parts.

**Review by ElsLommelen — COMMENTED** [review]
Already a minor comment on the changes itself.
But I prefer to wait with the final review until the docker is build. ;-)

**Review by ElsLommelen — APPROVED** [review]
Build successfully, so feel free to merge (after changing my suggestion?)

File: RELEASES.md (hunk 1 above)
Comment (hansvancalster) [reply RELEASES.md:27]: Good catch. Fixed in [e927edb](https://github.com/inbo/protocolsource/pull/73/commits/e927edb052968dd3fe13981295049b92693b48d9)

**Pull request merged**